fuse/subgraph.yaml

```diff
@@ -68,10 +68,32 @@ dataSources:
       language: wasm/assemblyscript
       entities:
         - Citizen
+        - TransactionStatistics
+        - Transfer
       abis:
         - name: GoodDollar
-          file: ./abis/gooddollar.abi
+          file: ./node_modules/@gooddollar/goodcontracts/build/contracts/GoodDollar.json
```

Comment:
i dont think this is correct because the .json is not the abi
the abi is a field inside that json

Reply:
it is handled correctly and it works

Reply:
i really dont see how. because it is simply not the ABI.
try to delete all the generated code and run codegen again, see if it works correctly.

Reply:
I did and it works
you can try it yourself, if it wouldn't work I will change it, it is not a big change

it's not such a big deal you implement such a feature